### ISSUE TITLE
Installed in Ubuntu  Ubuntu 20.04.2.0 LTS server version failed 

### ISSUE THREAD

**Issue by maoyang** [poseidon-network/qlauncher-linux#27]
my installation log 

[qlauncher][INFO] 2021-07-10T13:57:09Z  Start install k3s
 [OK] Install k3s
[qlauncher][INFO] 2021-07-10T13:57:25Z  Create service file
[qlauncher][INFO] 2021-07-10T13:57:25Z  systemd: Creating service file /etc/systemd/system/qlauncher.service
[qlauncher][INFO] 2021-07-10T13:57:25Z  systemd: Creating service env file /usr/local/bin/qlauncher.env
[qlauncher][INFO] 2021-07-10T13:57:25Z  systemd: Enabling qlauncher unit
[qlauncher][INFO] 2021-07-10T13:57:26Z  systemd: Starting qlauncher
Job for qlauncher.service failed because the control process exited with error code.
See "systemctl status qlauncher.service" and "journalctl -xe" for details.

**Comment by maoyang** [comment]
I executed systemctl status qlauncher.service the got the result 

qlauncher.service - qlauncher version 2
     Loaded: loaded (/etc/systemd/system/qlauncher.service; enabled; vendor preset: enabled)
     Active: activating (start) since Sat 2021-07-10 14:01:18 UTC; 478ms ago
    Process: 20598 ExecStartPre=/sbin/modprobe br_netfilter (code=exited, status=0/SUCCESS)
    Process: 20608 ExecStartPre=/sbin/modprobe overlay (code=exited, status=0/SUCCESS)
Cntrl PID: 20610 (qlauncher.sh)
      Tasks: 2
     Memory: 4.2M
     CGroup: /system.slice/qlauncher.service
             ├─20610 /bin/bash /opt/qlauncherV2/qlauncher.sh restart
             └─20686 /bin/sleep 10

Jul 10 14:01:18 swar systemd[1]: Starting qlauncher version 2...
Jul 10 14:01:18 swar qlauncher.sh[20618]: [qlauncher][INFO] 2021-07-10T14:01:18Z  Detect sudo/root user
Jul 10 14:01:18 swar qlauncher.sh[20618]: [qlauncher][INFO] 2021-07-10T14:01:18Z  [OK] User has sudo/root access right
Jul 10 14:01:18 swar qlauncher.sh[20618]: [qlauncher][INFO] 2021-07-10T14:01:18Z  Stop qlauncherV2

**Comment by coderkk** [comment]
I think maybe the issue with your network setting. Please re-check again your network. Thank you.

**Comment by maoyang** [comment]
Thanks for your online supporting ，I've resolved network DNS setting issue ，and re-installed qlauncher follow by github readme ，but I got error message when did this step 

/opt/qlauncherV2/qlauncher.sh status

then output

FATA[0010] failed to connect: failed to connect: context deadline exceeded

**Comment by coderkk** [comment]
Could you try uninstall qlauncher
`sudo /opt/qlauncherV2/qlauncher.sh uninstall`

Then run the install again
`sudo /opt/qlauncherV2/qlauncher.sh install`

**Comment by maoyang** [comment]
I've followed your steps but the result is same

**Comment by coderkk** [comment]
Please check this file ` /etc/crictl.yaml` exist or not?

If not exist, please try to create one.
```
# cat /etc/crictl.yaml
runtime-endpoint: unix:///run/containerd/containerd.sock
image-endpoint: unix:///run/containerd/containerd.sock
timeout: 2
debug: true
pull-image-on-create: false
```

Refer to this https://github.com/cri-o/cri-o/issues/1922

**Comment by maoyang** [comment]
yes the /etc/crictl.yaml didn't exist then I followed your steps and re-start qlauncher 

I execute the command -->sudo /opt/qlauncherV2/qlauncher.sh status

output 

DEBU[0000] ListContainerRequest: &ListContainersRequest{Filter:&ContainerFilter{Id:,State:nil,PodSandboxId:,LabelSelector:map[string]string{},},}
DEBU[0000] ListContainerResponse: nil
FATA[0000] listing containers failed: rpc error: code = Unimplemented desc = unknown service runtime.v1alpha2.RuntimeService

**Comment by coderkk** [comment]
This issue occur because of the compatible between k3s's containerd and dockerd.
